docs/ui/components/ContentSpotlight/index.tsx

```diff
@@ -0,0 +1,124 @@
+import { mergeClasses } from '@expo/styleguide';
+import dynamic from 'next/dynamic';
+import { useState } from 'react';
+import { type CSSProperties } from 'react';
+import VisibilitySensor from 'react-visibility-sensor';
+
+import { LightboxImage } from './LightboxImage';
+
+const ReactPlayer = dynamic(() => import('react-player/lazy'), { ssr: false });
+
+const PLAYER_WIDTH = '100%' as const;
+const PLAYER_HEIGHT = 400 as const;
+const YOUTUBE_DOMAINS = ['youtube.com', 'youtu.be'] as const;
+
+type ContentSpotlightProps = {
+  alt?: string;
+  src?: string;
+  url?: string;
+  file?: string;
+  caption?: string;
+  controls?: any;
+  spaceAfter?: boolean | number;
+  loop?: boolean;
+  style?: CSSProperties;
+  containerClassName?: string;
+};
+
+export function ContentSpotlight({
+  alt,
+  src,
+  url,
+  file,
+  caption,
+  controls,
+  spaceAfter,
+  loop = true,
+  style,
+  containerClassName,
+}: ContentSpotlightProps) {
+  const isYouTubeDomain = (url?: string) => {
+    return url ? YOUTUBE_DOMAINS.some(domain => url.includes(domain)) : false;
+  };
+
+  const getInitialMarginBottom = (spaceAfter: ContentSpotlightProps['spaceAfter']) => {
+    if (typeof spaceAfter === 'undefined') {
+      return 'mb-2';
+    } else if (typeof spaceAfter === 'number') {
+      return `mb-${spaceAfter}`;
+    } else if (spaceAfter) {
+      return 'mb-12';
+    }
+    return 'mb-0';
+  };
+
+  const [hover, setHover] = useState(false);
+  const [forceShowControls, setForceShowControls] = useState(isYouTubeDomain(url));
+  const isVideo = !!(url || file);
+
+  return (
+    <figure
+      className={mergeClasses(
+        'text-center py-2.5 my-5 rounded-lg',
+        containerClassName,
+        !isVideo && 'bg-subtle'
+      )}
+      onClick={() => {
+        if (typeof controls === 'undefined' && !forceShowControls) {
+          setForceShowControls(true);
+        }
+      }}
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      style={hover ? { cursor: 'pointer' } : undefined}>
```

Comment:
```suggestion
     >
```

Rewrite to Tailwind class.